MapboxVision/Services/Constants.swift

```diff
@@ -8,4 +9,6 @@ enum Constants {
     static let millisecondsInSecond: Double = 1000
     
     static let frameDuration: TimeInterval = 1 / 30.0
+
+    static let attitudeOrient: CMAttitudeReferenceFrame = .xArbitraryZVertical
```

Comment:
We don't need to have public constant here.
I suggest to have private enum in a local scope. Does that make sense?

Reply:
moved out here